src/sentry/seer/endpoints/group_ai_autofix.py

```diff
@@ -27,11 +38,42 @@
 logger = logging.getLogger(__name__)
 
 
+class AutofixRequestSerializer(CamelSnakeSerializer):
+    event_id = serializers.CharField(
+        required=False,
+        help_text="Run issue fix on a specific event. If not provided, the recommended event for the issue will be used.",
+    )
+    instruction = serializers.CharField(
+        required=False, help_text="Optional custom instruction to guide the issue fix process."
+    )
+    pr_to_comment_on_url = serializers.URLField(
+        required=False, help_text="URL of a pull request where the issue fix should add comments."
+    )
+    stopping_point = serializers.ChoiceField(
+        required=False,
+        choices=["root_cause", "solution", "code_changes", "open_pr"],
+        help_text="Where the issue fix process should stop. If not provided, will run to root cause.",
+    )
+
+
+class AutofixResponse(TypedDict):
+    """Response type for the POST endpoint"""
+
+    run_id: int
+
+
+class AutofixStateResponse(TypedDict):
+    """Response type for the GET endpoint"""
+
+    autofix: dict[str, Any] | None
```

Comment:
is it not possible to type more granularly than `dict[str, Any]`?

Reply:
I don't believe it is much added redundancy to add a typedict for this response type.

none of our endpoints use pydantic for their response serializers, so this is a bit of a one off.

Additionally, it seems like you modify the response state by removing fields and adding `repositories`. you'd need to modify the response type anyways.